### DISCUSSION TITLE
How could I add an email BCC to emails sent by supabase to users

### DISCUSSION THREAD

**Discussion by alexcdot** [supabase/supabase#21893] [Questions]
Hi all, I would like to add an additional email to be bcc'd whenever supabase sends an email to users about email confirmation, password resets, user invites, magic login links, etc. This is because we have users that complain they do not receive the emails, so we would like to sanity check that the email was sent (and maybe landed in the user's spam filter). It would be fine to have the same email apply for all supabase emails sent. Let me know if anyone has figured out a way to do this / or a workaround. We use sendgrid as our email provider.

**Comment by GaryAustin1** [comment]
You can create your own email sending serverside.  You use https://supabase.com/docs/reference/javascript/auth-admin-generatelink to get the links you need for your auth operations.  

You might also look at your SMTP provider and see what options they have for logging.

Finally you could request a feature enhancement to add a bcc field to the email processing in gotrue/auth.
